clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -3921,7 +3926,7 @@ void populateCIRToLLVMConversionPatterns(
       CIRToLLVMPtrMaskOpLowering,
       CIRToLLVMPtrStrideOpLowering,
       CIRToLLVMResumeOpLowering,
-      CIRToLLVMReturnAddrOpLowering,
+      CIRToLLVMFuncAddrBuiltinOpLowering,
```

Comment:
We should keep this list alphabetically sorted, to prevent duplicates from coming in. (That was part of @Lancern's motivation for the change.)